requirements.txt

```diff
@@ -3,3 +3,4 @@ django-crispy-forms==1.7.0
 django-debug-toolbar==1.9.1
 gunicorn==19.7.1
 psycopg2==2.7.3.2
+raven==6.6.0
```

Comment:
I don't see in this PR any sentry configuration, does this change relates to this PR?

Reply:
Removing this part.